src/content/advanced/security-policy-enforcement/policy-api.md

```diff
@@ -0,0 +1,69 @@
+---
+linkTitle: Giant Swarm Policy API
+title: Giant Swarm Policy API Guide
+description: An overview of how to use Giant Swarm Policy types to enforce cluster security and best practices.
+weight: 60
+menu:
+  main:
+    parent: advanced
+aliases:
+  - /advanced/security-policy-enforcement/policy-api
+user_questions:
+ -  How can I exclude a workload from a Kyverno policy?
+ -  What security policies are enforced in my cluster?
+ -  How do I migrate from PSPs to PSS?
+ -  What is the Policy API?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-shield
+last_review_date: 2023-10-04
+---
+
+<!-- {{< platform_support_table aws="alpha=v17.2.0" aws="ga=v17.4.0">}} -->
+
+__Note__: This guide is intended for cluster administrators running Giant Swarm managed Kubernetes clusters. More general information about Pod Security Standards can be found on the [Security policy enforcement][sec-policy-enforcement] page.
+
+## Managing cluster security policies with the Giant Swarm Policy API
+
+The Policy API is an abstraction layer which orchestrates several other types of policy-related resources.
+
+The Giant Swarm platform is built on top of a number of independent tools, projects, and APIs supported by the CNCF and the surrounding Kubernetes ecosystem.
+Among these are a number of resources designed for enforcing policies within a cluster. These include built-in types like Network Policies and RBAC as well as external CRDs like Kyverno Cluster Policies or Cilium Network Policies, among others.
+
+We use and manage tools which we see as bringing value for customers, and which are governed and maintained in a way which aligns with our guiding principles.
```

Comment:
Can we shorten this 3x"We" paragraph? Let's make it very explicit and use examples. I guess what we want to say here is that Kyverno and PSS is the current thing in CNCF landscape, but since it may change in the future, customers have the choice to use our high-level "I want app X to have restriction Y" policies over low-level Kyverno policies. We can then describe the benefits of our high-level CRDs as separate paragraph ("without requiring customers to change tools [...]").

What I mean is that this and the next paragraphs may be too verbose for a customer's engineer to read.

Reply:
Yeah I also don't like the We x 3, left it for future me in the next update.

This paragraph overall though is supposed to be general about the policy API. The current use cases and policies will be described in the lower sections, this is more to explain the motivation for even having the API and admittedly to put folks at ease about something so GS specific. Basically I'm toeing the line between wanting customers to use everything we provide and telling them to be realistic about what they want to manage